### ISSUE TITLE
About the seed

### ISSUE THREAD

**Issue by Ethanhuhuhu** [wengzejia1/Open-VCLIP#9]
Hi, @wengzejia1 ,

Thanks for your great work. Could you please share me the seed used for the B16 pretraininng? I test on UCF with the provided checkpoint and get 83.5 accuracy, but our re-implemented achieves only 82.4 and 82.6 after weight average. I think there might be some difference with the seed and could you help me? Thanks.

**Comment by wengzejia1** [comment]
Hi, do you use the same K400 dataset as ours (as illustrated in README.md), or your self-prepared K400 dataset?
What training script do you run?
And can you try different ``PATCHING_RATIO'' in the testing script to see if the accuracy will get better?

**Comment by Ethanhuhuhu** [comment]
Thanks for your reply.

I use the same K400 as yours, and there are about 10 videos get an error "failed to decode video idx *** :trial 0". I think it's not significant.

I use this script [https://github.com/wengzejia1/Open-VCLIP/blob/main/script/training/temporal_b16/train_tclip_interpolation_0.0-0.6_ratio0.5.sh](url).

And I've tested different patch ratio but still only achieve about 82.5 accuracy.

**Comment by wengzejia1** [comment]
Have you run the [SWA operation](https://github.com/wengzejia1/Open-VCLIP/blob/main/weight_average_tool.py) after you finished the training process ? That may influence the model performance.

By the way, I will rerun the training code to have a test.

**Comment by Ethanhuhuhu** [comment]
I ran the weight_average_tool and noticed a performance boost of 0.2. 

By the way, is the following error message important? 
‘failed to decode video idx *** :trial 0‘ 
I suspect that some videos might be missing, and I'm concerned if it will affect the accuracy.

**Comment by wengzejia1** [comment]
The error message ‘failed to decode video idx *** :trial 0‘ is not important. It is because a small amount of the download videos are broken.

**Comment by wengzejia1** [comment]
Hi, I rerun my code and the score is normal.
